### ISSUE TITLE
outliers correction 

### ISSUE THREAD

**Issue by Maroca5** [sgoldenlab/simba#339]
Hello, I'm using SLEAP H5 data for Simba, but when but when I run the outlier corrections it states is done but the document is empty 

![Screenshot](https://github.com/sgoldenlab/simba/assets/158096575/6d40642a-5473-4af0-8643-3f6f0ec014dc)

**Comment by sronilsson** [comment]
Hey @Maroca5! Thank you for reporting. Do you also have a screengrab of what is being printed out in the main SimBA terminal, and the Microsoft terminal, when this happens? It would help me to figure out what is going on.

**Comment by Maroca5** [comment]
hello, I don't have it sorry but now I'm running into another issue, Simba was using Python 3.10, I uninstall Python 3.10 to see if it will run with 3.6 but is not, is giving me an error [Fatal error in launcher: Unable to create process using '"C:\Users\Venere\AppData\Local\Programs\Python\Python310\python.exe"  "C:\Users\Venere\AppData\Local\Programs\Python\Python310\Scripts\simba.exe" ': The system cannot find the file specified.] how can I fix this?. thank you in advance.

**Comment by sronilsson** [comment]
Hi @Maroca5 - that looks a little odd, I don't know what `simba.exe` is. Did you install simba with `pip install simba-uw-tf-dev` ?

**Comment by Maroca5** [comment]
Hello again, so is installed with Anaconda, but is not finding ffmpeg [SimBA could not find a FFMPEG installation on computer (as evaluated by "ffmpeg" returning None). SimBA works best with FFMPEG and it is recommended to install it on your computer]
but it is installed 
![Screenshot (5)](https://github.com/sgoldenlab/simba/assets/158096575/59a55111-1ba1-429f-bde3-1e2d7bf379d9)

**Comment by Maroca5** [comment]
yes, 
![Screenshot (6)](https://github.com/sgoldenlab/simba/assets/158096575/0e38ab76-69f4-44e0-a587-4b437a0c7c89)

**Comment by Maroca5** [comment]
It runs it in the administrative and normal command prompt, but it doesn't on the anaconda terminal.

**Comment by sronilsson** [comment]
@Maroca5 - so I understand, you get the ffmpeg warning when you launch simba in the conda navigator terminal ?

**Comment by Maroca5** [comment]
figured it out Thank you, I installed it with anaconda and it worked, Thank you I will try to run a project hopefully I don't have any issues

**Comment by Maroca5** [comment]
Now when I'm trying to upload SLEAP files is giving me this error.
SIMBA NO DATA ERROR: No files with ['.slp'] extensions found in C:/Users/Venere/Desktop/MP4 videos new folder/Behavior.
I tried with .slp, H5 and CVS

**Comment by sronilsson** [comment]
What is the content of the `C:/Users/Venere/Desktop/MP4 videos new folder/Behavior` directory?

**Comment by Maroca5** [comment]
yes I moved all the SLEAP data there, when I first downloaded SIMBA with pip it will identify the files, only the H5, but now is not finding any of them.

**Comment by sronilsson** [comment]
What is the file extensions of the files in the `C:/Users/Venere/Desktop/MP4 videos new folder/Behavior` directory? Can you show me a screengrab?

**Comment by Maroca5** [comment]
![Screenshot (7)](https://github.com/sgoldenlab/simba/assets/158096575/540143a0-9c4f-4c48-a800-1b4e24fb138f)

**Comment by sronilsson** [comment]
Thanks @Maroca5 - that folder does not contain any tracking data, there are no files with h5, csv or slp extensions - I see 5 video MP4 files. That could be what SimBA is complaining about.  If they are located in a sub-folder to `C:/Users/Venere/Desktop/MP4 videos new folder/Behavior`, e,g, `:/Users/Venere/Desktop/MP4 videos new folder/Behavior/Behavior`, then try pointing SimBA to that directory.

**Comment by Maroca5** [comment]
I did and is still did work

**Comment by sronilsson** [comment]
And what are located in `C:/Users/Venere/Desktop/MP4 videos new folder/Behavior/Behavior` ?

**Comment by Maroca5** [comment]
so I have
![Screenshot (8)](https://github.com/sgoldenlab/simba/assets/158096575/47d25fe0-7be0-4d00-abfb-669d99366963)
 all the H5 files here

**Comment by Maroca5** [comment]
and I use this directory 
![Screenshot (9)](https://github.com/sgoldenlab/simba/assets/158096575/4e7fec3a-ae7b-4b61-b7f5-903cedb1c4af)

**Comment by sronilsson** [comment]
And just to confirm, what error do you see then?

**Comment by sronilsson** [comment]
Note: I would avoid having the best_model.h5 in that folder, because that is not a data file, and you don't want to try to import that as a data file.

**Comment by sronilsson** [comment]
Another note: .. the data files begin with `labels` prefix and are only 12kb large - just to check, its not your hand annotations in sleap is it? Or do the files contain machine generated body-part predictions for frames in the respective videos?

**Comment by Maroca5** [comment]
so it started to run when I took out the best-model but then it gave me this error
![Screenshot (10)](https://github.com/sgoldenlab/simba/assets/158096575/0a69f352-4f01-4041-a5f4-bdd15aa5acdc)

**Comment by sronilsson** [comment]
SimBA needs a way to pair your SLEAP video files, to your SLEAP data files. SimBA does this by looking at the filenames, `Video1.mp4` is paired with `Video1.h5` etc. In your case, your sleap h5 files is e.g., named `somethingveryverylong.h5`. Then SimBA searches for `somethingveryverylong.mp4` but can't find it, thats why you see this error:

<img width="1340" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/c08743db-1a2b-4d85-8e1b-29abf06a46ca">

Try renaming your sleap data to align with your video data file names.

**Comment by Maroca5** [comment]
It worked, but now the body parts don't match I'll fix that and continue, thank you so much for your help.

**Comment by Maroca5** [comment]
one last question, is there any way to do the analysis throughout simba, without having to use SLEAP or other software data? just doin everything with simba?

**Comment by sronilsson** [comment]
@Maroca5 you'd have to import it from another tool like SLEAP, or software that does pose estimation. I agree it would be good to have a basic pose estimation pipeline within simba but no plans for that at the moment
